bl/evaluation/printer.go

```diff
@@ -73,8 +74,12 @@ func PrintResults(resultsData *PrintResultsData) error {
 			return jsonOutput(&formattedOutput)
 		} else if resultsData.OutputFormat == "yaml" {
 			return yamlOutput(&formattedOutput)
-		} else {
+		} else if resultsData.OutputFormat == "xml" {
 			return xmlOutput(&formattedOutput)
+		} else if resultsData.OutputFormat == "JUnit" {
+			return jUnitOutput(&formattedOutput)
+		} else {
+			panic(errors.New("invalid output format"))
```

Comment:
The panic behavior is new,  it's ok by the product team?

Reply:
this should never ever happen (because it is validated way before) - so if it does, we will get notified via pagerduty

Reply:
BTW if you try to run datree test with -o that doesn't support we fail the run:
`Error: Invalid --output option - "blump"
Valid output values are - simple, yaml, json, xml, JUnit`